README.md

```diff
@@ -219,25 +225,23 @@ await foreach (var @event in client.ReadEventsAsync(
 
 ## Running EventQL Queries
 
-To run an EventQL query, call the `RunEventQlQueryAsync` method and provide the query as an argument. The method returns an async stream, which you can iterate over using `await foreach`:
+To run an EventQL query, call the `RunEventQlQueryAsync<TRow>` method and provide the query as an argument. The method returns an async stream, which you can iterate over using `await foreach`:
 
 ```csharp
 await foreach (var row in client.RunEventQlQueryAsync(
-    "FROM e IN events PROJECT INTO e"))
+    "FROM e IN events PROJECT INTO { count: COUNT() }"))
```

Comment:
Ich würde das Beispiel hier gerne beim ursprünglichen belassen. Kannst Du das bitte rückgängig machen?

Außerdem fehlt in dem Aufruf von `Run…` der generische Typparameter (müsste ja dann `Event` sein), oder übersehe ich da was?